pkgs/applications/virtualization/xen/generic/default.nix

```
@@ -390,7 +390,7 @@ stdenv.mkDerivation (finalAttrs: {
     ]
     ++ lib.lists.optional (!withInternalQEMU) "--with-system-qemu"
 
-    ++ lib.lists.optional withSeaBIOS "--with-system-seabios=${seabios}/share/seabios"
+    ++ lib.lists.optional withSeaBIOS "--with-system-seabios=${seabios.firmware}"
```

```suggestion
    ++ lib.lists.optional withSeaBIOS "--with-system-seabios=${seabios.firmware-location}"
```

This is a better name, given that seabios provide nothing besides a blob.

---

I named it `firmware` for parity with OVMF: https://github.com/NixOS/nixpkgs/blob/fe370c252d033f31e138582a8c5d4d8d2e3d8897/pkgs/applications/virtualization/OVMF/default.nix#L192

It looks pretty when they're close together, like in the Xen `with-system` options.